tephra-hbase-compat-0.96/src/test/java/co/cask/tephra/hbase96/coprocessor/TransactionProcessorTest.java

```diff
@@ -226,7 +226,11 @@ public void testDeleteFiltering() throws Exception {
       }
 
       // delete from the third entry back
-      Delete d = new Delete(row, V[5]);
+      long deleteTs = V[5] + 1;
```

Comment:
but in puts above it specifically skips putting into V[5], so you can write delete into it, no?